modules/activitypub/keypair_test.go

```diff
@@ -23,3 +28,40 @@ func TestKeygen(t *testing.T) {
 	assert.Regexp(t, regexp.MustCompile("^-----BEGIN PUBLIC KEY-----.*"), pub)
 
 }
+
+func TestSignUsingKeys(t *testing.T) {
+	priv, pub, err := GenerateKeyPair()
+	assert.NoError(t, err)
+
+	privPem, _ := pem.Decode([]byte(priv))
+	if privPem == nil || privPem.Type != "RSA PRIVATE KEY" {
+		t.Fatal("key is wrong type")
+	}
+
+	privParsed, err := x509.ParsePKCS1PrivateKey(privPem.Bytes)
+	if err != nil {
+		assert.NoError(t, err)
+	}
+
+	pubPem, _ := pem.Decode([]byte(pub))
+	if pubPem == nil || pubPem.Type != "PUBLIC KEY" {
+		t.Fatal("key failed to decode")
+	}
+
+	pubParsed, err := x509.ParsePKIXPublicKey(pubPem.Bytes)
+	if err != nil {
+		assert.NoError(t, err)
+	}
```

Comment:
```suggestion
	assert.NoError(t, err)
```

Reply:
Thanks, you are right that is indeed much better :)